src/MistyClosures.jl

```diff
@@ -1,5 +1,15 @@
 module MistyClosures
 
-# Write your package code here.
+using Core: OpaqueClosure
+using Core.Compiler: IRCode
+
+struct MistyClosure{Toc<:OpaqueClosure}
+    oc::Toc
+    ir::IRCode
+end
+
```

Comment:
Let's also add:
```julia
MistyClosure(ir::IRCode; kwargs...) = MistyClosure(OpaqueClosure(ir; kwargs....), ir)
```
Possibly even as the only inner constructor so we know that noone is giving us IR that doesn't match to the opaque closure

Reply:
Good idea

Reply:
I've not made it an inner constructor because I often need to recycle an existing OpaqueClosure -- obviously in this situation I'm taking responsibility for ensuring that this is done correctly.

I've added a note to the readme to indicate the safe + recommended way, vs the not recommend way.